make/examples.mk

```diff
@@ -20,6 +21,8 @@ examples.sync: kind.create clusterctl.init
       --kubernetes-version $(KUBERNETES_VERSION) \
       --control-plane-machine-count=1 \
       --worker-machine-count=1 | \
+    gojq --yaml-input --yaml-output \
+      '. | (select(.kind=="ClusterClass").spec.patches|= del(.[] | select(.name == "imageRepository")))' | \
```

Comment:
I like this, in the end we should be able to delete all patches and add our single external patch.